data/reporting-guide.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Reporting Guide
+header: Reporting Guide
+bloglike: true
+nav: reporting-guide
+---
+
+At PyCon India 2023, we are committed to providing a safe and welcoming
+environment for all attendees. We take incidents of harassment, misconduct, or
+violations of our [Code Of Conduct](../code-of-conduct) very seriously. This
+Reporting Guide aims to provide clear instructions on how to report any concerns
+and the steps we will take to address reported incidents.
+
+### Reporting Process
+
+If you experience any harassment, or unfair treatment, or witness any
+behavior you believe violates the PyCon India [Code Of Conduct](../code-of-conduct),
+we encourage you to report it as soon as possible. Reports can be made through the
+following channels:
+
+a. **Contact a Volunteer:** Look for individuals wearing "PyCon India Volunteer"
```

Comment:
@irangareddy This has to fixed. A Volunteer should not take any report from participants. A volunteer should redirect the participant to CoC workgroup area.

Reply:
@irangareddy Please go ahead, I already have confirmation from @inovizz. We have to publish this ASAP

Reply:
@Niveshkrishna, please validate.  Also, if everything seems good, this PR can be merged.